<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>设置对象</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_ListSource">
          <property name="text">
           <string>listSource</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_ListWidget">
          <property name="text">
           <string>listWidget</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_TreeWidget">
          <property name="text">
           <string>treeWidget</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_TableWidget">
          <property name="text">
           <string>tableWidget</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_dragDropMode">
       <property name="title">
        <string>拖放参数设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkBox_acceptDrop">
            <property name="text">
             <string>acceptDrops</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_dragEnable">
            <property name="text">
             <string>dragEnabled</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>dragDropMode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_DragDrop">
            <item>
             <property name="text">
              <string>NoDragDrop</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DragOnly</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DropOnly</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DragDrop</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>InternalMove</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_defaultDropAction">
            <property name="text">
             <string>defaultDropAction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_MoveAction">
            <item>
             <property name="text">
              <string>CopyAction</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MoveAction</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LinkAction</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IgnoreAction</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>listSource</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget_ListSource">
          <item>
           <property name="text">
            <string>粘贴</string>
           </property>
           <property name="icon">
            <iconset theme="audio-x-generic"/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>粗体</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>斜体</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>下划线</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左对齐</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中间对齐</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右对齐</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>红色</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </item>
          <item>
           <property name="text">
            <string>绿色</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </item>
          <item>
           <property name="text">
            <string>黄色</string>
           </property>
           <property name="background">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>159</red>
              <green>159</green>
              <blue>159</blue>
             </color>
            </brush>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>listWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="listWidget_ListWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>treeWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>tableWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
